src/main/java/de/metas/ui/web/window/datatypes/json/DateTimeConverters.java

```diff
@@ -138,18 +138,17 @@ else if (widgetType == DocumentFieldWidgetType.Timestamp)
 		}
 	}
 
-	private static Timestamp fromJdbcTimestampToLocalDateTime(final Object valueObj)
+	private static Timestamp fromJdbcTimestampToLocalDateTime(@NonNull final String s)
 	{
-		return Timestamp.valueOf((String)valueObj);
+		return Timestamp.valueOf(s);
 	}
 
 	/*
 	 * Saved User Query records generate the date as jdbc timestamp, and these fail for the formats we use.
 	 * They appear as follows: `2016-06-11 00:00:00.0`.
```

Comment:
irrelevant Javadoc.
instead explain what actually this method is doing,
i.e. try to guess if a given string has JDBC format, so it's parsable by Timestamp.valueOf(String).
Returns true in case the given string is in JDBC format. IMPORTANT: possible does not mean 100% sure
Returns false when the given string is NOT in JDBC format. IMPORTANT: this is 100% sure.

PS: feel free to better formulate it.

Reply:
updated javadoc.